api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -24,9 +24,24 @@ message KubernetesObjectData {
   oneof kind_specific {
     KubernetesPodData pod = 100;
     KubernetesNodeData node = 101;
+    KubernetesServiceData service = 102;
   }
 }
 
+message NodePorts {
+  // The port on each node on which this service is exposed
+  int32 node_port = 1;
+  // The port that will be exposed by this service
+  int32 port = 2;
+  // The IP protocol for this port, supports "TCP", "UDP", and "SCTP"
+  string protocol = 3;
+}
+
+message KubernetesServiceData {
+  repeated string ip_addresses = 1;
+  repeated NodePorts node_ports = 2;
```

Comment:
Add another field for `LoadBalancer` ports? I think we shouldn't mix them with `NodePort`s.
Or better, just keep this named `ports` as it's named `ports` in the k8s `spec`. That would be more consistent.
```suggestion
  repeated ServicePorts ports = 2;
```